spec/helpers/application_helper_spec.rb

```diff
@@ -20,4 +20,12 @@
       end
     end
   end
+
+  describe "#markdown" do
+    context "converts to markdown" do
+      it "when given valid markdown" do
+        expect(helper.markdown("_foo_")).to eq("<p><em>foo</em></p>\n")
+      end
```

Comment:
This test was passing around lunchtime. Like, for realz. And yet now it fails consistently on my box and on the CI build complaining about the trailing newline character.

I "fixed" the test by adding the trailing newline to the expected result, but meh.